presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -355,3 +360,75 @@ Lambda Aggregate Functions
         -- (2, 42)
 
     The state type must be a boolean, integer, floating-point, or date/time/interval.
+
+
+Filtering During Aggregation
+----------------------------
+
+The ``FILTER`` keyword can be used to remove rows from aggregation processing
+with a condition expressed using a ``WHERE`` clause. This is evaluated for each
+row before it is used in the aggregation and is supported for all aggregate
+functions.
+
+    AGGREATE_FUNCTION(...) FILTER (WHERE <condition>)
+
+A common and very useful example is to use ``FILTER`` to remove nulls from
+consideration when using ``array_agg``.
+
+.. code-block:: none
+
+    SELECT array_agg(col) FILTER (WHERE col IS NOT NULL)
+    FROM t
+
+As another example, imagine you want to add a condition on the count for Iris
+flowers, modifying the following query.
+
+.. code-block:: none
+
+    SELECT species,
+           count(*) AS count
+    FROM memory.default.iris
+    GROUP BY species;
+
+.. code-block:: none
+
+    species    | count
+    -----------+-------
+    setosa     |   50
+    virginica      50
```

Comment:
The formatting is off here. This row is missing `|` and the next one is misaligned